### DISCUSSION TITLE
Using Spatie and Sanctum

### DISCUSSION THREAD

**Discussion by begueradj** [spatie/laravel-permission#1882] [General]
[Laravel Sanctum ](https://laravel.com/docs/8.x/sanctum) utilizes Laravel's `web` authentication guard just as this Spatie package does.  
Given this fact, when I use Sanctum to authenticate my API, are there any considerations I should take ? I am asking this question following what you stated on your [official documentation](https://spatie.be/docs/laravel-permission/v5/basic-usage/multiple-guards) when using a guard other than this one.

Thank you.
Billal BEGUERADJ

**Comment by erikn69** [comment]
Many use sanctum or passport without problems, just follow the documentation

**Comment by ersin-demirtas** [comment]
You can always use multiple guards for your api endpoints and you are probability using with sanctum guard "auth:sanctum". Just remember that you can attach multiple guards "auth:psk,sanctum". The most important clue is the $guard_name, you must declare it in the model you are using to authorise the request.
